### PR TITLE
Jose

### PR DESCRIPTION
**Pull request by jcortes89**
Fixed lab 2, completed lab 4 and fixed 5. Working on 6 and will finish soon. Decided to restart it due to me coding my way into a corner and making it far too complicated for me to understand anymore haha :'D Thank you!

### CODE REVIEW

File: code/jose/Python_Labs/lab02.py [hunk 1]
```diff
@@ -5,8 +5,27 @@
 # hundreds_digit = x//100
 # print(hundreds_digit)
```
Comment (brucestull) [inline code/jose/Python_Labs/lab02.py:7]: [Lab 2: Number to Phrase](https://github.com/PdxCodeGuild/class_swordfish/blob/main/1%20Python/labs/02%20Number%20To%20Phrase.md)

**Needs Work**

* Input of `104`:
    * Expecter output: `one hundred four`
    * Actual output:
    ```
    PS C:\Users\Bruce\Programming\class_swordfish\code\jose\Python_Labs> python .\lab02.py
    Enter a number: 94
    ninety four
    PS C:\Users\Bruce\Programming\class_swordfish\code\jose\Python_Labs> python .\lab02.py
    Enter a number: 104
    Traceback (most recent call last):
      File "C:\Users\Bruce\Programming\class_swordfish\code\jose\Python_Labs\lab02.py", line 82, in <module>
        print(hundreds_digit + 'hundred',tens_digitwritten[tens_digit],ones[ones_digit])
    KeyError: 0
    PS C:\Users\Bruce\Programming\class_swordfish\code\jose\Python_Labs>
    ```

File: code/jose/Python_Labs/lab04.py [hunk 2]
```diff
@@ -13,7 +13,9 @@ def pick6():
     return numbers
```
Comment (brucestull) [inline code/jose/Python_Labs/lab04.py:13]: [Lab 4: Pick6](https://github.com/PdxCodeGuild/class_swordfish/blob/main/1%20Python/labs/04%20Pick%206.md)

**Completed**

**Comment by brucestull** [comment]
[Lab 5: Credit Card Validation](https://github.com/PdxCodeGuild/class_swordfish/blob/main/1%20Python/labs/05%20Credit%20Card%20Validation.md) has already been completed.

**Completed**

**Review by brucestull — CHANGES_REQUESTED** [review]
Lab 2 needs work.

**Comment by jcortes89** [comment]
just closing to reopen a new one to pull my lab 7 into the branch since it is completed

**Pull request closed without merging**